### ISSUE TITLE
Any modern documents?

### ISSUE THREAD

**Issue by GF-Huang** [sestoft/C5#95]
Statement of the question.

**Comment by siriak** [comment]
What do you mean?

**Comment by LevYas** [comment]
Yeah, it'd also be nice to have links to the documentation and examples. I found this https://www.itu.dk/research/c5/Release1.0/c5doc/types/C5.TreeDictionary%602.htm , but it's for version 1, found this https://c5docs.azurewebsites.net/annotated.html but it does not contain the TreeDictionary. The book is nice, but it's terribly outdated.
